django/thunderstore/community/models/package_listing.py

```diff
@@ -91,13 +91,21 @@ def __str__(self):
     def get_absolute_url(self):
         return reverse(
             "packages.detail",
-            kwargs={"owner": self.package.owner.name, "name": self.package.name},
+            kwargs={
+                "owner": self.package.owner.name,
+                "name": self.package.name,
+                "community_identifier": self.community.identifier,
+            },
         )
 
     @cached_property
-    def owner_url(self):
+    def get_owner_url(self):
```

Comment:
No need to prefix this with `get_` since we no longer take in any parameters and thus can keep this a property.